### ISSUE TITLE
Invalid API key

### ISSUE THREAD

**Issue by thedigitallion** [patcon/flashfood-api-docs#2]
Hi,

Is this code dead? I've tried to work with it but when I play with the API on Swagger, it says Unauthorized.

**Comment by Nekenlight** [comment]
no, you have to put the access token in the api call that you do :

curl -X POST https://identity.flashfood.com/oauth/token -d @payload.json --header "Content-Type: application/json"

get access_token then : 

curl -X 'GET' \
  'https://cors-anywhere.herokuapp.com/https://api.flashfood.com/api/v1/store/list?food_images=1&latitude=xxxxxxx&longitude=xxxxxxxxxxx&distance=xxxxxx&show_all=1' \
  -H 'accept: */*' \
  -H 'Authorization: Bearer **XXXXXXX**'

**Comment by patcon** [comment]
Oh hey, didn't know anyone was using this! Glad if it's helpful..! Also, gives me a warm open sourcey feeling that someone else hopped in to answer a question 😊🙏🏻

**Comment by patcon** [comment]
Am I ok to close this? Happy to accept a pill request to improve docs, if you think some change might've made it easier to help yourself @thedigitallion :)

**Comment by thedigitallion** [comment]
I actually did figure it out in the end, as no one responded for quite a while. Thank you so much for this git! It's been a huge help!
